wiki/Tournaments/Common_seeding_methods/en.md

```diff
@@ -0,0 +1,182 @@
+
+### Rounding
+
+Unless otherwise specified, it is important to **avoid intermediate rounding** (unless for display purposes only). For example, if the Percent Maximum seeding method is being used, each of the `Map points` should not be rounded to the nearest hundredth before being added together, since this rounding may alter teams’ relative ranking. In seeding methods like Z-Sum or Z-Percentile, it is common for different teams’ total scores to look the same when rounded but actually be slightly different; in those cases, the true numbers, rather than the rounded ones, should be used.
+
+### Weighting maps
+
+Many of the methods above assign each team a number of points on each map and then add or average those points together to determine final seeding. It may also make sense to perform a weighted average so that some maps contribute more than others to final standings. For example, osu!mania tournaments may assign niche skill sets or physically demanding maps a lower weight than fundamental skill sets or accuracy-based challenges.
+
+More precisely, a positive number for each map, called the `Map weight`, should be specified. The final number used for seeding is then
+
+![Weighting formula](img/weight.png)[^mappool-size]
+
+where `Map points` is the quantity that the seeding method assigns to each map (e.g. `Map placement` for Average Placement). Note that having all map weights be equal to `1` or `1/(Number of maps)` would just be equivalent to summing or averaging the map points together.
+
+## Differences between methods
+
+There are many factors that go into choosing a seeding method for a tournament. First of all, the following principles are important to keep in mind:
+
+- **Lack of qualifiers**: All of the seeding methods above assume that teams all play through the same qualifiers pool. If qualifiers are not played, then it may make sense instead to seed by rank or [BWS](/wiki/Tournaments/Badge-weighted_seeding). (For example, a 2v2 tournament with team size 4 may seed teams by the sum of their two best BWS ranks, with the smallest sum being seeded 1st.)
+  - Note that some tournaments may begin head-to-head competition by placing teams into a group stage or Swiss-system tournament instead of a standard elimination bracket. However, these other systems still require specification on how the teams are placed into matchups, so often some method of seeding is still required (either one of the ones mentioned in this page, or random drawings).
+- **Scoring system**: Most of the seeding methods above are typically used when tournaments are played with the ScoreV2 scoring system. Thus, extra care should be taken when seeding tournaments that use ScoreV1, accuracy, or other methods of scoring. For example, a map played with ScoreV1 may have high outliers from well-performing teams, so it may be less advised to seed by Z-Percentile or Percent Maximum.
```

Comment:
```suggestion
- **Scoring system**: Most of the seeding methods above are typically used when tournaments are played with the ScoreV2 scoring system. Thus, extra care should be taken when seeding tournaments that use ScoreV1, accuracy, or other methods of scoring. For example, a map played with ScoreV1 may have high outliers from well-performing teams, so it may be less appropriate to seed by Z-Percentile or Percent Maximum.
```